Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/InputActionsEditorSettingsProvider.cs

```diff
@@ -225,9 +268,33 @@ private void BuildUI()
 
         private InputActionAsset GetAsset()
         {
-            var serializedAsset = new SerializedObject(newAsset);
-            m_ActionEditorAnalytics.RegisterReset();
-            m_State = new InputActionsEditorState(m_ActionEditorAnalytics, serializedAsset);
+            return m_State.serializedObject?.targetObject as InputActionAsset;
+        }
+
+        private void SetObjectFieldEnabled(bool enabled)
+        {
+            // Update object picker enabled state based off editor play mode
+            if (m_RootVisualElement != null)
+                UQueryExtensions.Q<ObjectField>(m_RootVisualElement, "current-asset")?.SetEnabled(enabled);
+        }
+
+        private void ModeChanged(PlayModeStateChange change)
```

Comment:
This flow change seems separate from Analytics but perhaps I'm missing something.

Its tidy so I'm not too worried but these extra changes make the analytics PR bigger as it drifts from its focus.

Reply:
Diff should no longer exist, marking resolved